src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestGetCallGen.generated.txt

```diff
@@ -0,0 +1,101 @@
+// <auto-generated/>
+#nullable enable
```

Comment:
Curious, do we already have precedence for that? How do we enforce such a LangVer requirement?

Reply:
e.g. https://github.com/dotnet/runtime/blob/6cde942cb1ac97bd53915f843d1499da977647d5/src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs#L274-L280

Reply:
Thanks but I should have added "... in an out-of-band source generator".

Reply:
Just checked.  Both our System.Text.Json and Logging generators end up generating invalid code in a new .NET Framework project, because they both generate code that requires nullable, which is only available with C# 8+.

I opened:
https://github.com/dotnet/runtime/issues/83392
https://github.com/dotnet/runtime/issues/83393